markup5ever/util/buffer_queue.rs

```diff
@@ -49,6 +49,12 @@ pub struct BufferQueue {
     buffers: VecDeque<StrTendril>,
 }
 
+impl Default for BufferQueue {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
```

Comment:
Same comment here about removing the `new()` function.

Reply:
Done.